src/main/java/org/jabref/logic/importer/fileformat/CffImporter.java

```diff
@@ -92,73 +103,100 @@ public CffIdentifier() {
     public ParserResult importDatabase(BufferedReader reader) throws IOException {
         ObjectMapper mapper = new ObjectMapper(new YAMLFactory());
         CffFormat citation = mapper.readValue(reader, CffFormat.class);
-        HashMap<Field, String> entryMap = new HashMap<>();
-        StandardEntryType entryType = StandardEntryType.Software;
-
-        // Map CFF fields to JabRef Fields
-        HashMap<String, Field> fieldMap = getFieldMappings();
-        for (Map.Entry<String, String> property : citation.values.entrySet()) {
-            if (fieldMap.containsKey(property.getKey())) {
-                entryMap.put(fieldMap.get(property.getKey()), property.getValue());
-            } else if ("type".equals(property.getKey())) {
-                if ("dataset".equals(property.getValue())) {
-                    entryType = StandardEntryType.Dataset;
-                }
-            } else if (getUnmappedFields().contains(property.getKey())) {
-                entryMap.put(new UnknownField(property.getKey()), property.getValue());
-            }
-        }
 
-        // Translate CFF author format to JabRef author format
-        String authorStr = citation.authors.stream()
-                        .map(author -> author.values)
-                        .map(vals -> vals.get("name") != null ?
-                                new Author(vals.get("name"), "", "", "", "") :
-                                new Author(vals.get("given-names"), null, vals.get("name-particle"),
-                                        vals.get("family-names"), vals.get("name-suffix")))
-                        .collect(AuthorList.collect())
-                        .getAsFirstLastNamesWithAnd();
-        entryMap.put(StandardField.AUTHOR, authorStr);
-
-        // Select DOI to keep
-        if ((entryMap.get(StandardField.DOI) == null) && (citation.ids != null)) {
-            List<CffIdentifier> doiIds = citation.ids.stream()
-                            .filter(id -> "doi".equals(id.type))
-                            .collect(Collectors.toList());
-            if (doiIds.size() == 1) {
-                entryMap.put(StandardField.DOI, doiIds.getFirst().value);
-            }
+        StandardEntryType entryType = StandardEntryType.Misc;
+
+        if (citation.type != null) {
+            entryType = mapType(citation.type);
         }
+        BibEntry entry = new BibEntry(entryType);
+        HashMap<Field, String> entryMap = new HashMap<>();
 
-        // Select SWHID to keep
```

Comment:
I don't see this in the new code.

Isn't there a test case for that? Yes -- org.jabref.logic.importer.fileformat.CffImporterTest#importEntriesSwhIdSelect1